example/lib/main.dart

```diff
@@ -237,6 +238,14 @@ class _MyHomePageState extends State<MyHomePage> {
               source: Text('9'),
             ),
           ),
+          PaneItemSeparator(),
+          PaneItemAction(
+            icon: const Icon(FluentIcons.open_source),
+            title: const Text('Source code'),
+            onTap: () {
+              launch('https://github.com/bdlukaa/fluent_ui');
+            },
+          ),
```

Comment:
Isn't it better on `footerItems`?

Reply:
I hesitated, honestly I don't know, both seem similar. Do you have a preference?

1. 
![image](https://user-images.githubusercontent.com/8223773/143771135-59771573-b37f-4917-8a27-5049f3f8ab62.png)

2. 
![image](https://user-images.githubusercontent.com/8223773/143771150-1b8691c2-7d95-4473-9b34-00211bff59b1.png)

3. 
![image](https://user-images.githubusercontent.com/8223773/143771164-26781fad-9e23-42bd-aac0-04082c5bb003.png)

4. 
![image](https://user-images.githubusercontent.com/8223773/143771181-67c0dbde-0171-498d-a990-d338b2dd3bcb.png)

Reply:
2 is the best IMHO